k-distribution/tests/regression-new/set-symbolic-tests/failing/union-5-spec.k.out.actual

```diff
@@ -0,0 +1,22 @@
+  #Ceil ( S
+  ( SetItem ( Y:MyId ) -Set SetItem ( X:MyId ) ) )
+#And
+  <k>
+    unionResult ( S
+    SetItem ( X:MyId )
+    ( SetItem ( Y:MyId ) -Set SetItem ( X:MyId ) ) ) ~> .
+  </k>
```

Comment:
This result is partly due to rewriting `|Set` into a `-Set` , but it is not correct.
We started with `(S SetItem(X)) |Set SetItem(Y)` and  `Y` could be in `S`, making the resulting term undefined.
Of course the `#Ceil` condition will (cum grano salis) prevent that but... it is a valid case that is dropped by the simplification.
Many (removed) `union` tests have this same problem.